{"name": "test_login_with_valid_email_and_invalid_password_Duplicate", "status": "passed", "start": 1695710993196, "stop": 1695710996094, "uuid": "8ff20400-6dee-4f38-bb78-d5f0b588656a", "historyId": "9581a2f8ea7f6829aae721e266b65c65", "testCaseId": "9581a2f8ea7f6829aae721e266b65c65", "fullName": "Tests.test_Login(duplicate).TestPanarayLoginDuplicate#test_login_with_valid_email_and_invalid_password_Duplicate", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup_and_teardown', 'log_on_failure')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login(duplicate)"}, {"name": "subSuite", "value": "TestPanarayLoginDuplicate"}, {"name": "host", "value": "WONIT11L32CV6Z"}, {"name": "thread", "value": "21668-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login(duplicate)"}]}